util/promote_staging_data

```diff
@@ -63,9 +66,6 @@ while getopts "ht:alp" OPTION; do
   esac
 done
 
-# Default to dry run if promotion is not selected
-DRY_RUN_ARG=${DRY_RUN_ARG:--n}
```

Comment:
Had to remove this because if `DRY_RUN_ARG=""`, it'll interpret it as empty and set it to `-n`. So `-n` was always selected whether or not promote data was selected.

Reply:
:woman_facepalming: woops